### PR TITLE
DEVELOPER-3213 Migration of search to DCP registered query

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

**Comment by LightGuard** [comment]
Has this been tested with Drupal / export ?

**Comment by adelasofia** [comment]
@LightGuard it has not

File: javascripts/search.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ search.service('searchService',function($http, $q) {
       query_highlight: true
     });
 
-    var endpoint = (!!window.location.pathname.match(/\/search/) ? app.dcp.url.search : app.dcp.url.developer_materials);
+    var endpoint = app.dcp.url.developer_materials;
```
Comment (LightGuard) [inline javascripts/search.js]: Where is this variable located?

Comment (adelasofia) [inline javascripts/search.js]: @lightguard I'm not exactly sure what you mean, but this variable is used on the next line (#20)

**Comment by adelasofia** [comment]
retest this please

**Pull request merged**